languageserver/src/main/scala/langserver/messages/Commands.scala

```diff
@@ -247,15 +247,9 @@ object Initialized {
 
 case class CancelRequest(id: Int) extends Notification
 
-case class FileEvent(uri: String, `type`: Int)
+case class FileEvent(uri: String, tpe: FileChangeType)
```

Comment:
Will s/`type`/tpe/ produce the same JSON?

Reply:
Good catch! I changed for consistency with `ShowMessageParams` and `LogMessageParams` above, not realizing that this one is actually serialized. I will change it back and move to `types.scala`

Reply:
By the way, those mentioned `...Params` should also have `type` field. I think this was the reason that the warning messages were shown as info-type notifications in Atom (I think in VSCode as well).. So it's a really good catch 👌